style.css

```diff
@@ -332,61 +259,150 @@ img.placeholder {
 .lang-wrapper {
   padding-top: 100px;
   display: flex;
-  gap: 7px;
+  gap: 5px;
   justify-content: center;
-  align-items: center;
 }
 
-.lang-wrapper-1,
-.lang-wrapper-2 {
-  padding-top: 30px;
+.language-list {
   display: flex;
-  flex-direction: row;
-  margin-top: 5px;
+  list-style: none;
+  text-align: center;
+  flex-direction: column;
+  align-items: center;
   justify-content: center;
-  gap: 7px;
+  margin-top: 5px;
+  width: 100%;
+}
+
+.framework {
+  padding-top: 15px;
+}
+
+/* contact us information */
+.contact {
+  background-color: #fff;
+  display: flex;
+  display: grid;
```

Comment:
[OPTIONAL] You can remove the display flex as the display grid would be used since it is the latter.

Reply:
thank you I will